packages/muelu/cmake/MueLu_config.hpp.in

```diff
@@ -108,6 +108,11 @@
 
 #cmakedefine HAVE_MUELU_MAGMASPARSE
 
+
+#cmakedefine HAVE_MUELU_HYPRE
```

Comment:
Don't we need a corresponding line in CMake? Or do we want to use `-D HAVE_MUELU_HYPRE`?